docs/usage/verticles/ModelRefreshVerticle.md

```diff
@@ -0,0 +1,26 @@
+# ModelRefreshVerticle
+
+The `ModelRefreshVerticle` is a Vert.x verticle that watches a specified directory for changes and reloads the
+models when a change is detected. It can be configured to trigger a model refresh in response to different types of
+events, such as the creation or modification of a file in the watched directory. The `ModelRefreshVerticle` also has a
+configurable check interval to specify how often it should check for changes in the directory. It defaults to 5 seconds.
+
+To use the `ModelRefreshVerticle`, you will need to provide it with a directory to watch and ensure that it is
+deployed in your Vert.x application. You can also customize the check interval and event types by providing a different
+interval and/or setting the `watchLogic` configuration flag.
+
+## Configuration
+
+The following options can be set to configure the `ModelRefreshVerticle`.
+
+| Property     | Type   | Required | Description                                                                                                                 | Default |
+|--------------|--------|:--------:|-----------------------------------------------------------------------------------------------------------------------------|---------|
+| `watchLogic` | string |    No    | Whether to listen only on `CREATE`, or also on `MODIFY` events. If both should be enabled, the value must be set to `copy`. | `~`     |
```

Comment:
I think it is not possible to have both

Reply:
Maybe we should simply mention that this verticle extends WatchVerticle and therefore the watch logic can be set and then link to the section in WatchVerticle